extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -341,6 +344,12 @@ export class ConsentUI {
 
     this.resetAnimationStyles_();
 
+    this.viewer_.sendMessage(
```

Comment:
I think we should use the `Viewport.enterLightboxMode()` instead?

Reply:
So `enterLightboxMode()` does a lot more than I think is neccessary. [see my response here](https://github.com/ampproject/amphtml/issues/22994#issuecomment-514381509) to @jridgewell 😄

Reply:
I do see that it handles fixed layer and and FIE. 

In our case, the FIE won't be an issue because the request can never be from a component in FIE. But I find it's doesn't hurt to include it. 
Not sure about fixed layer. @jridgewell could you please help us understand if the message is enough here instead of calling `Viewport.enterLightBoxMode()`. Thanks.

Reply:
If it's an already fixed position element, it might not need any special fixed-layer support?

Reply:
Yeah I don't think it requires any special fixed-layer support 😄

Reply:
Sounds good. Thanks!